Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -552,6 +554,7 @@ extension ActivityStreamPanel {
                 site.setBookmarked(true)
             })
         }
+        self.telemetry.reportEvent(.Bookmark, source: pingSource, position: index)
```

Comment:
Also, you would want to duplicate this event - one for removing and another for adding a bookmark.